### ISSUE TITLE
Callback issue

### ISSUE THREAD

**Issue by mronoffon** [jorgebastida/django-dajaxice#34]
Dajaxice/ JS isn't 'connecting' to ajax.py from the admin.

**Comment by jorgebastida** [comment]
Hi mronoffon,

I'm not sure, but probably the problem is that your admin isn't loading the dajaxice.core.js, hope this help you: [1]

``` python
class YourModelAdmin(admin.ModelAdmin):
    class Media:
        js = ("/dajaxice/dajaxice.core.js",)

admin.site.register(YourModel, YourModelAdmin)
```

[1] https://docs.djangoproject.com/en/dev/ref/contrib/admin/#modeladmin-media-definitions

Cheers

**Comment by mronoffon** [comment]
Hi jorgebastida,

Actually, the dajaxice.core.js in class Media 'is' being loaded.  The problem is that any function from ajax.py is never found as in in my js file ( 'serverside_req' in Dajaxice.gallery.serverside_req).   I'm trying to use dajaxice in the admin pages but am not exactly sure 'how' ajax.py is found even when there's no use of urls.py.  I did, however, see that 'dajaxice_autodiscover' finds the ajax.py file by reviewing code/tracebacks.  Also, when i statically generated my .core.js file, i see that the 'gallery' property was added the code : 
        gallery: {
               callback_myexample: function(callback_function, argv){
                alert("REACHED Dajaxice.gallery.callback_myexample...");
                Dajaxice.call('gallery.callback_myexample', callback_function, argv);  
                //Dajaxice.call('callback_myexample', callback_function, argv);  
   ...

How do i add the property in the request to the server, what is the proper syntax?  

thanks in advance
